diff -ur ProofGeneral-4.2.orig/bin/proofgeneral ProofGeneral-4.2/bin/proofgeneral
--- ProofGeneral-4.2.orig/bin/proofgeneral	2013-11-05 20:12:01.260699915 +0000
+++ ProofGeneral-4.2/bin/proofgeneral	2013-11-05 20:12:18.030699612 +0000
@@ -75,10 +75,10 @@
 if [ -z "$PGHOME" ] || [ ! -d "$PGHOME" ]; then
     # default relative to this script, otherwise PGHOMEDEFAULT
     MYDIR="`readlink --canonicalize "$0" | sed -ne 's,/bin/proofgeneral$,,p'`"
-    if [ -d "$MYDIR" ]; then
-	PGHOME="$MYDIR"
-    elif [ -d "$PGHOMEDEFAULT" ]; then
+    if [ -d "$PGHOMEDEFAULT" ]; then
 	PGHOME="$PGHOMEDEFAULT"
+    elif [ -d "$MYDIR" ]; then
+	PGHOME="$MYDIR"
     else
 	echo "Cannot find the Proof General lisp files: Set PGHOME or use --pghome."
 	exit 1
